modules/claims_api/lib/bgs_service/veteran_representative_service.rb

```diff
@@ -1,19 +1,58 @@
 # frozen_string_literal: true
 
-require_relative 'veteran_representative_service/create_veteran_representative_request'
-require_relative 'veteran_representative_service/read_all_veteran_representatives'
-
 module ClaimsApi
   class VeteranRepresentativeService < ClaimsApi::LocalBGS
-    private
+    def bean_name
+      'VDC/VeteranRepresentativeService'
+    end
+
+    def create_veteran_representative(options)
+      # error VetRep cannot be null
+      injected = convert_nil_values(options)
 
-    def make_request(namespace:, **args)
-      super(
-        endpoint: 'VDC/VeteranRepresentativeService',
-        namespaces: { namespace => '/data' },
-        transform_response: false,
-        **args
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <data:VetRepresentative>
+          #{injected}
+        </data:VetRepresentative>
+      EOXML
+
+      make_request(
+        endpoint: bean_name,
+        namespaces: { 'data' => '/data' },
+        action: 'createVeteranRepresentative',
+        body: body.to_s,
+        key: 'VeteranRepresentativeReturn',
+        transform_response: false
       )
     end
+
+    # type_code: form type (I.E. 21-22 vs 21-22A)
+    # ptcpnt_id: participant ID
+    def read_all_veteran_representatives(type_code:, ptcpnt_id:)
+      validate! type_code, ptcpnt_id
+
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <ns0:CorpPtcpntIdFormTypeCode>
```

Comment:
we don't append an `ns0` namespace to our outgoing request so this fails with the 
```
<error><message>VetRep cannot be null</message></error>
```